web/cypress/integration/route/create-route-both-use-uri-uris.spec.js

```diff
@@ -0,0 +1,171 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable no-undef */
+
+context('Create Route Both use uri and uris', () => {
+  const selector = {
+    empty: '.ant-empty-normal',
+    name: '#name',
+    description: '#desc',
+    hosts_0: '#hosts_0',
+    hosts_1: '#hosts_1',
+    uris_0: '#uris_0',
+    uris_1: '#uris_1',
+    remote_addrs_0: '#remote_addrs_0',
+    remote_addrs_1: '#remote_addrs_1',
+    nodes_0_host: '#nodes_0_host',
+    nodes_0_port: '#nodes_0_port',
+    nodes_0_weight: '#nodes_0_weight',
+    nameSelector: '[title=Name]',
+    drawer: '.ant-drawer-content',
+    monacoScroll: '.monaco-scrollable-element',
+    addHost: '[data-cy=addHost]',
+    addUri: '[data-cy=addUri]',
+    addRemoteAddr: '[data-cy=addRemoteAddr]',
+    deleteAlert: '.ant-modal-body',
+    notificationCloseIcon: '.ant-notification-close-icon',
+    notification: '.ant-notification-notice-message',
+  };
+
+  const data = {
+    name: 'test_route',
+    description: 'desc_by_autotest',
+    host: '12.12.12.12',
+    port: '80',
+    weight: 1,
+    host_0: '0.0.0.0',
+    uri_0: '/test',
+    remote_addr_0: '192.168.1.0',
+    host_1: '1.1.1.1',
+    uri_1: '/test1',
+    remote_addr_1: '192.168.1.1',
+    submitSuccess: 'Submit Successfully',
+    deleteRouteSuccess: 'Delete Route Successfully',
+  };
+
+  beforeEach(() => {
+    cy.login();
+  });
+
+  it('should create route with uri/host/remote_addr', () => {
+    cy.visit('/');
+    cy.contains('Route').click();
+    cy.get(selector.empty).should('be.visible');
+    cy.contains('Create').click();
+    cy.contains('Next').click().click();
+    cy.get(selector.name).type(data.name);
+    cy.get(selector.description).type(data.description);
+
+    cy.get(selector.hosts_0).type(data.host_0);
+    cy.get(selector.uris_0).type(data.uri_0);
+    cy.get(selector.remote_addrs_0).type(data.remote_addr_0);
+
+    cy.contains('Next').click();
+    cy.get(selector.nodes_0_host).type(data.host);
+    cy.get(selector.nodes_0_port).type(data.port);
+    cy.get(selector.nodes_0_weight).type(data.weight);
+    cy.contains('Next').click();
+
+    cy.contains('Next').click();
+    cy.contains('Submit').click();
+    cy.contains(data.submitSuccess);
+
+    // back to route list page
+    cy.contains('Goto List').click();
+    cy.url().should('contains', 'routes/list');
+  });
+
+  it('Raw Data should have uri/host/remote_addr', () => {
+    cy.visit('/');
+    cy.contains('Route').click();
+
+    cy.get(selector.nameSelector).type(data.name);
+    cy.contains('Search').click();
+    cy.contains(data.name).siblings().contains('More').click();
+    cy.contains('View').click();
+    cy.get(selector.drawer).should('be.visible');
+
+    cy.get(selector.monacoScroll).within(() => {
+      cy.contains('uri').should('exist');
```

Comment:
It would be better to  verify the corresponding values are correct

Reply:
I didn't find a way to get the value through the field in the `Row Data Editor`, any good suggestions?

In addition, is it possible to verify that the values are equal on the configuration page?

Reply:
Done ~